src/content/keyAutoAdd/lib/keyAutoAdd.ts

```diff
@@ -121,24 +121,19 @@ export class KeyAutoAdd {
     return promise;
   }
 
-  private async run({
-    walletAddressUrl,
-    publicKey,
-    nickName,
-    keyId,
-    keyAddUrl,
-  }: BeginPayload) {
-    const params: StepRunParams = {
-      walletAddressUrl,
-      publicKey,
-      nickName,
-      keyId,
-      keyAddUrl,
+  private async run(payload: BeginPayload) {
+    const helpers: StepRunHelpers = {
```

Comment:
Can `helpers` be a private property of the class, since it doesn't depend on anything? This way we are not going to create this variable every time we call `run` throughout the auto-add flow.

Reply:
Run is called only on page reload kinda.

Reply:
The other function called `run` in this method threw me off.